moninag/static/src/ts/app/user-profile/user-profile.component.html

```diff
@@ -1,53 +1,24 @@
-        <div class="row">
-            <div class="col-xs-12 col-sm-12 col-md-6 col-lg-6 col-xs-offset-0 col-sm-offset-0 col-md-offset-3 col-lg-offset-3">
-            <h2>Hi {{user.first_name}}!</h2>
-        </div>
-    </div>
-
-    <div class="row">
-        <div class="col-md-4  toppad  pull-right col-md-offset-7 ">
-            <a href="auth/logout/">Logout</a>
-        </div>
-        <div class="col-xs-12 col-sm-12 col-md-6 col-lg-6 col-xs-offset-0 col-sm-offset-0 col-md-offset-3 col-lg-offset-3 toppad" >
-
-            <div class="panel panel-info">
-                <div class="panel-heading">
-                    <h3 class="panel-title">{{ user.last_name }}</h3>
-                </div>
-
-                <div class="panel-body">
-                    <div class="row">
-                        <div class="col-md-3 col-lg-3 " align="center"> <img alt="User Pic" src="https://www.gravatar.com/avatar/{{ user.avatar }}" class="img-circle img-responsive"> </div>
-                            <div class=" col-md-9 col-lg-9 ">
-                                <table class="table table-user-information">
-                                    <tbody>
-                                        <tr>
-                                            <td>Name:</td>
-                                            <td>{{ user.first_name }}</td>
-                                        </tr>
-                                        <tr>
-                                            <td>Surname:</td>
-                                            <td>{{ user.second_name }}</td>
-                                        </tr>
-
-                                        <tr>
-                                            <td>Email: </td>
-                                            <td>
-                                                <a href="mailto:{{ user.email }}">{{ user.email }}</a>
-                                            </td>
-                                        </tr>
-                                    </tbody>
-                                </table>
-                            </div>
-                        </div>
-                    </div>
-                </div>
-            </div>
-        </div>
-    </div>
+<div *ngIf="user" class="flex-container" fxLayout="row" fxLayoutAlign="center start">
+    <img alt="User picture" src="https://www.gravatar.com/avatar/{{ user.avatar }}">
+    <table>
+        <tbody>
+            <tr>
+                <td>Name:</td>
+                <td>{{ user.first_name }}</td>
+            </tr>
+            <tr>
+                <td>Surname:</td>
+                <td>{{ user.second_name }}</td>
+            </tr>
+            <tr>
+                <td>Email: </td>
+                <td>
+                    <a href="mailto:{{ user.email }}">{{ user.email }}</a>
+                </td>
+            </tr>
+        </tbody>
+    </table>
 </div>
 <div *ngIf="errorMessage">
-    {{errorMessage}}
+    {{ errorMessage }}
 </div>
```

Comment:
Good idea, but I will do it in separate task, for now I just remove bootstrap styles and add flex layout, so it looks readable :upside_down_face: